ui/app/selectors/selectors.js

```diff
@@ -86,7 +86,7 @@ export const getMetaMaskAccounts = createSelector(
 )
 
 export function getSelectedAddress (state) {
-  const selectedAddress = state.metamask.selectedAddress || Object.keys(getMetaMaskAccounts(state))[0]
+  const selectedAddress = state.metamask.selectedAddress
 
   return selectedAddress
```

Comment:
nit: why not return this directly instead?

```suggestion
  return state.metamask.selectedAddress
```

Reply:
Yes, good idea. I meant to - I forgot to make this change. I had just pushed this up run it on CI initially.